File: Python/1043.py
(a, b, c) = map(float, input().split(" "))
if(a < (b + c) and b < (c + a) and c < (a + b)):
    print("Perimetro = %.1f" % (a + b + c))
else:
    print("Area = %.1f" % (((a + b) * c) / 2))

File: Javascript/1043.js
var input = `7.0 5.0 7.0`;
let [a, b, c] = input.split(/[\s]/).map((x) => Number(x));

if (a < b + c && b < c + a && c < a + b) {
  console.log("Perimetro = " + (a + b + c).toFixed(1));
} else {
  console.log("Area = " + (((a + b) * c) / 2).toFixed(1));
}
